src/ts/src/BowlingGame.test.ts

```diff
@@ -0,0 +1,14 @@
+import { BowlingGame } from './BowlingGame';
+
+describe('BowlingGame', () => {
+	it('Test boilerplate that fails', () => {
```

Comment:
I thought it could be a good idea to leave these 3 lines as a kind of a boilerplate for reference, can delete them if this is not necessary